packages/common/src/credentialDeploymentTransactions.ts

```diff
@@ -177,3 +182,35 @@ export function getAccountAddress(credId: string): AccountAddress {
     );
     return accountAddress;
 }
+
+export type CredentialInputV1 = {
+    ipInfo: IpInfo;
+    globalContext: CryptographicParameters;
+    arsInfos: Record<string, ArInfo>;
+    idObject: IdentityObjectV1;
+    revealedAttributes: AttributeKey[];
+    seed: string;
```

Comment:
We could improve the naming here so that it is clear what format the seed has, e.g. `seedAsHex` or `seedAsMnemonic`.